src/components/Layout/Layout.tsx

```diff
@@ -318,7 +325,10 @@ export const Layout = ({
               </View>
               <View key={asPathWithNoHash} className="layout-main">
                 <Flex
+                  id={mainId}
                   as="main"
+                  tabIndex={-1}
```

Comment:
This tabIndex allows the element to have focus styles when navigated to it by the SkipToMain link. Otherwise when you're just normal keyboard navigating it won't receive focus.